website/src/docs/uppy.md

```diff
@@ -372,6 +372,9 @@ uppy.addFile({
   name: 'my-file.jpg', // file name
   type: 'image/jpeg', // file type
   data: blob, // file blob
+  meta: {
+      relativePath: string, // optional, determines the unique relative path to build the file uppy id
```

Comment:
The file ID is kind of a detail. I think the important thing for end-users is that it allows you to add multiple identical files. Maybe we can do something like this?
```suggestion
    // optional, store the directory path of a file so Uppy can tell identical files in different directories apart
    relativePath: webkitFileSystemEntry.relativePath,
```